### ISSUE TITLE
[Bug]: Non-silenced notifications

### ISSUE THREAD

**Issue by Anonim19992** [flatpak/flatpak#5533]
### Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/flatpak/flatpak/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/flatpak/flatpak/issues) for a bug that matches the one I want to file, without success.
- [X] If this is an issue with a particular app, I have tried filing it in the appropriate issue tracker for the app (e.g. under https://github.com/flathub/) and determined that it is an issue with Flatpak itself.
- [X] This issue is not a report of a security vulnerability (see [here](https://github.com/flatpak/flatpak/blob/main/SECURITY.md) if you need to report a security issue).

### Flatpak version

1.15.4

### What Linux distribution are you using?

Manjaro Linux

### Linux distribution version

Linux anton-ms7a34 6.5.3-1-MANJARO #1 SMP PREEMPT_DYNAMIC Wed Sep 13 12:21:35 UTC 2023 x86_64 GNU/Linux

### What architecture are you using?

x86_64

### How to reproduce

When entering `flatpak update | grub flatout 2> /dev/null | wc -l`, some of the information is not silenced, although there is an explicit "2> /dev/null" 
An application that requires an earlier runtime

### Expected Behavior

Only the number of rows in which `flatpak update | grub flatout 2> /dev/null | wc -l` is given is given.

### Actual Behavior

flatpak update | grep flathub 2> /dev/null | wc -l
error: org application.gnome.Calculator/x86_64/stable requires an org runtime environment.gnome.Platform/x86_64/45, which is not installed
1
![2023-09-25-142307_956x518_scrot](https://github.com/flatpak/flatpak/assets/43230744/f3058f5b-d3c6-4815-bb20-34b2a330e05c)

**Comment by StayPirate** [comment]
This is not a bug, I guess you meant to run `flatpak update 2>/dev/null | grep flathub | wc -l`.

**Comment by Anonim19992** [comment]
т9 =) flatpak update 2>/dev/null does not drown out information, etc. messages from flatpak. 
`[anton@anton-ms7a34 ~]$ flatpak update 2>/dev/null

Поиск обновлений…

Информация: среда выполнения org.kde.Platform ветвь 6.4 конец срока поддержки по причине:
   We strongly recommend moving to the latest stable version of the Plaform and SDK
Информация: приложения, использующие эту среду выполнения:
   com.obsproject.Studio

Информация: (закрепленная) среда выполнения org.freedesktop.Platform ветвь 21.08 конец срока поддержки по причине:
   org.freedesktop.Platform 21.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Информация: приложения, использующие эту среду выполнения:
   net.xmind.XMind

Информация: среда выполнения org.freedesktop.Platform.GL.default ветвь 21.08 конец срока поддержки по причине:
   org.freedesktop.Platform 21.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Информация: приложения, использующие это расширение:
   net.xmind.XMind

Нечего выполнять.
`

**Comment by Anonim19992** [comment]
It would be convenient if there was a tool (for example, pamac checkupdate, but for flatpak) that would check only for certain updates.

**Comment by StayPirate** [comment]
Yes, because these messages are printed to `stdout`. I agree that these specific messages should be outputed to `stderr`. In my case [I get the proper list of available updates](https://github.com/StayPirate/dotfiles/blob/71072d79bba842a5090a24824e523b1dbb19f70d/.config/waybar/config#L338-L350) via the following regex:

```bash
flatpak update | grep -E "^ ?[0-9]{1,2}\." | wc -l
```

**Comment by Anonim19992** [comment]
> Yes, because these messages are printed to `stdout`. I agree that these specific messages should be outputed to `stderr`. In my case [I get the proper list of available updates](https://github.com/StayPirate/dotfiles/blob/71072d79bba842a5090a24824e523b1dbb19f70d/.config/waybar/config#L338-L350) via the following regex:
> 
> ```shell
> flatpak update | grep -E "^ ?[0-9]{1,2}\." | wc -l
> ```

Alas, but it is equivalent to flatpak update | grub flathub 2> /dev/null | wc -l =(Something like that. I will wait for the withdrawal to be made via stderr.

**Comment by hfiguiere** [comment]
Flathub is more of a TUI than a pure CLI. It's not mean to be parsed, but rather read by humans. So the EOL warning definitely belong to stdout. It's not an error, it's an output result.

**Comment by bbhtt** [comment]
I can't read Russian, but if the initial screenshot is about the EOL runtime message then it is intentional that it goes to stdout.
